### PR TITLE
1970 more component playgrounds

### PR DESCRIPTION
**Pull request by GCHQ-Developer-530**
## Summary of the changes
Add playground stories for chip, classification banner, data entity, dialog, empty state and toast.
Update dismissMode on toast so that it can change between manual and automatic

## Related issue
Part of #1970 

### General 

- [x] Changes to docs package checked and committed.
- [x] All acceptance criteria reviewed and met. 

### Testing

- [x] Relevant unit tests and visual regression tests added. 
- [x] Visual testing against Figma component specification completed. 

### Accessibility 

- [x] Accessibility Insights FastPass performed.
- [x] A11y unit test added and yields no issues.
- [x] A11y plug-in on Storybook yields no issues. 
- [x] Manual screen reader testing performed using NVDA and VoiceOver. 
- [x] Manual keyboard testing for keyboard controls and logical focus order. 
- [x] Correct roles used and ARIA attributes used correctly where required. 
- [x] Logical heading structure is maintained, and the HTML elements used for headings can be changed to fit within the wider page structure. 

### Resize/zoom behaviour 

- [x] Page can be zoomed to 400% with no loss of content. 
- [x] Screen magnifier used with no issues. 
- [x] Text resized to 200% with no loss of content.
- [x] Text spacing increased as per the [WCAG 1.4.12 success criterion](https://www.w3.org/TR/WCAG21/#text-spacing) with no loss of content.

### System modes

- [x] Browser setting 'prefers reduced motion' tested. No animations or motion visible whilst this setting is on. 
- [x] Windows High Contrast mode tested with no loss of content.
- [x] System light and dark mode tested with no loss of content.
- [x] Browser support tested (Chrome, Safari, Firefox and Edge). 

### Testing content extremes

- [x] Min/max content examples tested with no loss of content or overflow. 
- [x] All prop combinations work without issue. 
- [x] Tested for FOUC (Flash of Unstyled Content) in both SSR (Server-Side Rendering) and SSG (Static Site Generation) settings.
- [x] Controlled and uncontrolled input components tested.

### CODE REVIEW

File: packages/web-components/src/components/ic-toast/ic-toast.stories.mdx [hunk 1]
```diff
@@ -341,3 +341,90 @@ import readme from "./readme.md";
...
+    parameters={{ loki: { skip: true } }}
+    argTypes={{
+      dismissMode: {
+        options: ["manual", "automatic"],
+        control: {
+          type: "inline-radio",
+        },
+      },
+      variant: {
+        options: ["neutral", "info", "warning", "error", "success"],
```
Comment (GCHQ-Developer-847) [inline packages/web-components/src/components/ic-toast/ic-toast.stories.mdx:371]: I think this needs an empty string option too as the variant prop is optional so is allowed to be undefined as well (like in the Default story)

File: packages/react/src/stories/ic-toast.stories.mdx [hunk 2]
```diff
@@ -337,3 +337,89 @@ import readme from "../../../web-components/src/components/ic-toast/readme.md";
...
+    parameters={{ loki: { skip: true } }}
+    argTypes={{
+      dismissMode: {
+        options: ["manual", "automatic"],
+        control: {
+          type: "inline-radio",
+        },
+      },
+      variant: {
+        options: ["neutral", "info", "warning", "error", "success"],
```
Comment (GCHQ-Developer-847) [inline packages/react/src/stories/ic-toast.stories.mdx:367]: Same point here as with the web components toast playground

File: packages/react/src/stories/ic-dialog.stories.mdx [hunk 3]
```diff
@@ -790,3 +790,59 @@ export const showAutoOpenCloseButtonDialog = () => {
...
+  buttons: true,
+  closeOnBackdropClick: true,
+  destructive: false,
+  dismissLabel: "Dismiss",
+  disableHeightConstraint: false,
+  disableWidthConstraint: false,
+  hideCloseButton: false,
+  heading: "This is a dialog",
+  label: "Dialog",
+  open: false,
```
Comment (GCHQ-Developer-847) [inline packages/react/src/stories/ic-dialog.stories.mdx:806]:
```suggestion
  open: true,
```
As per my comment in the web components playground

**Review by GCHQ-Developer-847 — CHANGES_REQUESTED** [review]
A few small comments

Also getting some weird behaviour with the toast playground when trying to change between the manual and automatic dismiss mode, but I think this is just a Storybook problem - all looks fine in the actual stories, so all good

**Review by GCHQ-Developer-847 — APPROVED** [review]

**Review by GCHQ-Developer-465 — APPROVED** [review]

**Pull request merged**